packages/jmix-front-generator/src/generators/react-typescript/app/template/i18n-message-packs/zh-cn.json

```diff
@@ -3,7 +3,8 @@
   "management.browser.delete.ok": "删除",
   "management.browser.noItems": "暂无数据",
   "management.editor.validationError": "验证错误。请检查输入的数据",
-  "management.editor.success": "实体已更新",
+  "management.editor.created": "实体已创建",
```

Comment:
Typically we don't update languages we are not certain of (en/ru). We do not use translation services as it may result in absolutely incorrect wording